server/roomlogs.ts

```diff
@@ -117,7 +118,7 @@ export class Roomlog {
 		this.modlogStream = stream;
 		this.sharedModlog = true;
 	}
-	async setupRoomlogStream(sync = false) {
+	async setupRoomlogStream() {
```

Comment:
> `Roomlog#setupRoomlogStream` has been changed to only be async,
> because it was using async functions so it was never really
> "sync" even if the sync param was true.

I don't care; the `sync` parameter gets around a race condition. Call it `partialSync` if you think the name is misleading, but please don't introduce bugs for purity reasons.

Reply:
> I don't care; the sync parameter gets around a race condition. Call it partialSync if you think the name is misleading, but please don't introduce bugs for purity reasons.

+1